Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -613,6 +613,10 @@ def audit_homepage
       problem "The homepage should start with http or https (URL is #{homepage})."
     end
 
+    unless homepage =~ %r{^(.+)://(.*)/(.*)}
+      problem "#{homepage} must have a slash after the domain name part."
```

Comment:
I agree, but I had URLs like `https://example.com#anchor` or `https://exampe.com?page=100` in mind. These are valid and missing a slash, but not from the end.